operators/bosh-operator/BoshOperator.js

```diff
@@ -71,7 +71,7 @@ class BoshOperator extends BaseOperator {
         resourceId: _.get(changeObjectBody, 'metadata.name'),
         status: {
           response: response,
-          state: CONST.APISERVER.RESOURCE_STATE.IN_PROGRESS
+          state: _.get(response, 'task_id') ? CONST.APISERVER.RESOURCE_STATE.IN_PROGRESS : CONST.APISERVER.RESOURCE_STATE.WAITING
```

Comment:
waiting is not a valid state that can be returned back to CC from broker. Just checking. I hope we still send `in progress` when request is queued right?

Reply:
As discussed, this is required for staggered deployment watching.

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok

Reply:
ok.